pkg/tracker/balance.go

```diff
@@ -19,17 +19,17 @@ import (
 const BalanceTrackerName = "BalanceTracker"
 
 type BalanceTracker struct {
-	db      db.DB
-	client  rpc.ETHClient
-	account tellorCommon.Account
+	db      *db.DB
+	client  *rpc.ETHClient
```

Comment:
```suggestion
	db      db.DB
	client  rpc.ETHClient
```
these are interfaces so no need to save as pointers